src/blocks/card.css

```
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background-color: var(--color-white);
+  border: 1px solid var(--main-border-color);
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--text-main-color);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+}
+
+.card__review-box {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-bottom: 24px;
+}
```

The same BEM problem.
```suggestion
.card__review-box {
  margin-bottom: 24px;
}

.review-box {
  display: flex;
  justify-content: space-between;
  align-items: flex-end;
}
```